src/cephadm/cephadm.py

```diff
@@ -2036,6 +2036,19 @@ def get_fqdn():
     return socket.getfqdn() or socket.gethostname()
 
 
+def get_ip_addresses(hostname: str) -> Tuple[List[str], List[str]]:
+    ipv4_addresses = []
+    ipv6_addresses = []
+    for res in socket.getaddrinfo(hostname, None,
+                                  flags=socket.AI_CANONNAME,
+                                  type=socket.SOCK_STREAM):
+        if res[0] == socket.AF_INET:
+            ipv4_addresses.append(res[4][0])
+        elif res[0] == socket.AF_INET6:
+            ipv6_addresses.append(res[4][0])
+    return ipv4_addresses, ipv6_addresses
```

Comment:
Maybe I'm too fan of Python list comprehensions but...
```suggestion
    items = socket.getaddrinfo(hostname, None,
                               flags=socket.AI_CANONNAME,
                               type=socket.SOCK_STREAM):   
    ipv4_addresses = [i[4][0] for i in items if i[0] == socket.AF_INET]
    ipv6_addresses = [i[4][0] for i in items if i[0] == socket.AF_INET6]
    return ipv4_addresses, ipv6_addresses
```

Reply:
> Maybe I'm too fan of Python list comprehensions but...

The resulting code is shorter and readable, so why not :+1: